packages/material/src/controls/MaterialEnumControl.tsx

```diff
@@ -37,12 +37,19 @@ import { MaterialInputControl } from './MaterialInputControl';
 import { MuiAutocomplete, WithOptionLabel } from '../mui-controls/MuiAutocomplete';
 
 export const MaterialEnumControl = (props: ControlProps & OwnPropsOfEnum & WithOptionLabel) => {
-  const {config, uischema} = props;
+  const {config, uischema, errors, visible} = props;
   const appliedUiSchemaOptions = merge({}, config, uischema.options);
+  const isValid = errors.length === 0;
   return (
+    appliedUiSchemaOptions.autocomplete === false ?
     <MaterialInputControl
       {...props}
-      input={appliedUiSchemaOptions.autocomplete === false ? MuiSelect : MuiAutocomplete}
+      input={MuiSelect}
+    />:
+    <MuiAutocomplete
+      {...props}
+      isValid={isValid}
+      visible={visible}
```

Comment:
* `visible` is already handed over via the `...props` so we don't need to do this explicitely.
 * There is no need to determine and hand over `isValid` here. Just do the calculation within the `MuiAutocomplete`.